src/main.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "library"
+
+def main
+  lib = Library.new("../storage.yml")
```

Comment:
Propose to create `storage.yml` from `seeds.rb` If it's not present

Reply:
fixed